### PR TITLE
[ENH] Extract graphs from impact page

### PR DESCRIPTION
**Pull request by iamdamion**
made dashboards section a new page. 

Screen Shot 2024-12-05 at 9 15 00 AM
Screen Shot 2024-12-05 at 9 16 20 AM

### CODE REVIEW

**Comment by christinerogers** [comment]
- [ ] change page title `Dashboards` :  `Measuring BIDS Impact`
- [ ] for the Gender of contributors, add `as of June 2024`

File: docs/impact/dashboards.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Dashboards
+## Measuring BIDS Impact
```
[the remark this answers is not in the document]
Comment (bendhouseart) [reply docs/impact/dashboards.md]: Possibly rename files to `measuring_BIDS_impact.md`

File: mkdocs.yml [hunk 2]
```diff
@@ -169,6 +169,7 @@ nav:
     -   Using MkDocs macros: extensions/macros_doc.md
 -   Impact:
     -   impact/index.md
+    -   impact/dashboards.md
```
Comment (bendhouseart) [inline mkdocs.yml:172]:
```suggestion
    -   impact/measuring_BIDS_impact.md
```

Rename to make rendering/reading of markdown yaml more clear later.

**Review by bendhouseart — CHANGES_REQUESTED** [review]
Looks good, just need to remove/change first heading/title, requested changes added with code suggestions/comments.

**Comment by Remi-Gau** [comment]
> * for the Gender of contributors, add `as of June 2024`

Will do that in a follow up PR to make sure this is automatically updated when recount this when we have new contributors.

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**